### PR TITLE
Bootstrapped Django 3.1.

### PR DESCRIPTION
**Pull request by felixxm**
(to be merged after the stable/3.0.x branch is created following Django 3.0 alpha)

### CODE REVIEW

**Comment by timgraham** [comment]
The commits for each feature removal should also add an entry under "Features removed in 3.1" in the release notes.

**Comment by felixxm** [comment]
Thanks Tim for reminding me :+1:

File: django/contrib/staticfiles/storage.py [hunk 1]
```diff
@@ -473,20 +471,6 @@ def hash_key(self, name):
         return 'staticfiles:%s' % key
 
 
-class CachedStaticFilesStorage(CachedFilesMixin, StaticFilesStorage):
```
Comment (felixxm) [inline django/contrib/staticfiles/storage.py]: I think that we should also remove `CachedFilesMixin` and `_MappingCache` that are currently unused. (\cc @carltongibson).

Comment (carltongibson) [reply django/contrib/staticfiles/storage.py]: Yes, absolutely.

**Comment by felixxm** [comment]
@carltongibson Thanks :+1:

**Pull request merged**